### DISCUSSION TITLE
iconPrefix

### DISCUSSION THREAD

**Discussion by tzwtmll** [vuepress-theme-hope/vuepress-theme-hope#3082] [Q&A] [answered]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I followed the docs and I double checked my configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/) and I did not find the answer.

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [ ] macOS
- [X] Windows
- [ ] Linux

### Describe the bug

![image](https://user-images.githubusercontent.com/104382132/235124847-f1d2e2c6-a281-4232-8693-8e25fb2f65a7.png)
![image](https://user-images.githubusercontent.com/104382132/235125123-affe7434-b139-46f5-b01d-2f39a864a731.png)

### Additional context

icon- 我直接俄写到md文件的icon中了，是完整的，这个没问题，但是为什么他是一个框

**Comment by tzwtmll** [comment]
![image](https://user-images.githubusercontent.com/104382132/235125641-d831b0db-70d4-4afa-a642-46bb98e1425e.png)

**Comment by tzwtmll** [comment]
![image](https://user-images.githubusercontent.com/104382132/235125709-30ee5ce6-a1ec-4ea7-b970-b295a3cded26.png)

**Comment by Mister-Hope** [comment, marked as answer]
Read the docs plz, we automatically set iconPrefix for you.
